playkit/src/main/java/com/kaltura/playkit/plugins/KalturaLiveStatsPlugin.java

```diff
@@ -159,6 +142,26 @@ public void onEvent(PKEvent event) {
                     case PLAYBACK_INFO_UPDATED:
                         PlaybackInfo currentPlaybackInfo = ((PlayerEvent.PlaybackInfoUpdated) event).getPlaybackInfo();
                         lastReportedBitrate = currentPlaybackInfo.getVideoBitrate();
+                        isLiveStream = currentPlaybackInfo.getIsLiveStream();
+                        break;
+                    case SOURCE_SELECTED:
+                        PlayerEvent.SourceSelected sourceSelected = (PlayerEvent.SourceSelected) event;
+                        switch (sourceSelected.sourceExtention) {
```

Comment:
@giladna, @noamtamim if we expose a source object we can add a helper function on it to translate this